recipe/meta.yaml

```
@@ -15,16 +15,15 @@ package:
 source:
   git_url:  https://github.com/pyg-team/pyg-lib/
   git_rev: {{ version }}
-
   patches:
     - fix-cmake.patch
 
 build:
-  number: 4
+  number: 0
   string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
   string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
   # Only build for CUDA >=11.2
```

Sorry missed this comment needs an update with the `skip` change

```suggestion
  # Only build for CUDA >=11.8
```